Cargo.toml

```diff
@@ -19,6 +19,7 @@ hex = "0.4.3"
 num = { version = "0.4.0", features = ["serde"] }
 uuid = { version = "0.8.2", features = ["serde"]}
 bigdecimal = {version ="0.3.0", features = ["serde"]}
+serde = { version = "1.0.111", features = ["derive"] }
```

Comment:
Why this specific version of `serde`? We can use the latest.

Reply:
It doesn't matter.
That will still evaluate to the latest 1.x.y release as it's still semver compatible

Reply:
Sure, but why specify this version if we don't need to? Anyhow not going to block on this as it's pretty pedantic.

Reply:
No problems. Once you rebase we can get this merged.

Reply:
If you want to release this, open a PR changing the version in `Cargo.toml`. Then tag the commit when it's merged.